physijs/README.md

```diff
@@ -0,0 +1,18 @@
+# PhysiJS Changes
+
+The files here are 99.9% the same as the original [`physijs`](https://github.com/chandlerprall/Physijs) code.
+There are a few small modifications to make it compatible with bundlers and work out-of-the-box with zero config:
+
+- accepting the `PhysijsWorker` constructor function as an argument instead of just using `new Worker(...)`
```

Comment:
should this link to the [MDN docs for `Worker()`](https://developer.mozilla.org/en-US/docs/Web/API/Worker/Worker)?

I think, like the Ammo/Emscripten stuff, my original reasoning of not linking so rabbit holes aren't given makes sense, but I do link to `importScripts` below...

Reply:
like the Ammo/Emscripten stuff, think I'm gonna leave it out, at least for now, and just go with my original reasoning for leaving it out. don't want to hold this back simply because of this consideration